docs/source/project_create.rst

```diff
@@ -53,31 +52,38 @@ values later in the *Details* page.
 Data
 ====
 
-In Step 2, you import a dataset and select prior knowledge.
-
-Add Dataset
+Add dataset
 -----------
 
 Click on *Add* to select a dataset. The data needs to adhere to a
-:doc:`specific format <data>`.
-
-Depending on the :ref:`Project mode <project_create:Project modes>`, you are
-offered the following options for adding a dataset. Keep in mind that in Oracle
-mode, your dataset is unlabeled or :ref:`data_labeled:Partially labeled data`. For Exploration and Simulation mode, you need :ref:`data_labeled:Fully labeled
-data`.
+:doc:`specific format <data>`. Keep in mind that in
+Oracle mode, your dataset is unlabeled or :ref:`data_labeled:Partially
+labeled data`; in Validation mode :ref:`data_labeled:Partially labeled data` or
+fully labeled; and for Simulation mode, you need :ref:`data_labeled:Fully
+labeled data`.
 
 .. tip::
 
     You will benefit most from what active learning has to offer with :ref:`data:High-quality data`.
 
+Depending on the :ref:`Project mode <project_create:Project modes>`, you are
+offered different options for adding a dataset:
 
 From File
-~~~~~~~~~
+---------
```

Comment:
I think the previous hierarchy is correct, and "From File" is part of the Add Dataset section.